### DISCUSSION TITLE
为什么我暂停视屏后再次播放页面的背景图就变成了黑色的,是因为我多点了哪个插件吗?

### DISCUSSION THREAD

**Discussion by yj200151** [the1812/Bilibili-Evolved#4651] [提问 / Q&A]
![Snipaste_2024-02-25_21-40-04](https://github.com/the1812/Bilibili-Evolved/assets/64696891/a14b2a6c-9f0b-4803-85ed-aad077637729)
![Snipaste_2024-02-25_21-40-27](https://github.com/the1812/Bilibili-Evolved/assets/64696891/be9a3ac6-88cd-4bac-8572-efefdc9d8a73)

**Comment by Geeyun-JY3** [comment]
`播放时自动关灯`
